static/bidder-info/adhese.yaml

```diff
@@ -1,4 +1,4 @@
-endpoint: "https://ads-{{.AccountID}}.adhese.com/json"
+endpoint: "https://ads-{{.AccountID}}.adhese.com/openrtb2"
```

Comment:
@mefjush  endpoint is not reachable

```
curl -i --location --request POST 'https://ads-1.adhese.com/openrtb2'
curl: (60) SSL certificate problem: self signed certificate
More details here: https://curl.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
```

Reply:
This is related to the other concern you already shared. We require the users of the adapter to specify the `AccountID` so it matches their account name in Adhese. An example endpoint that could be reached is  `https://ads-demo.adhese.com/openrtb2` - that's coupled with `demo` account in our system.